app/knights/knight.py

```diff
@@ -0,0 +1,30 @@
+from app.knights.armors import Armour, Weapon, Potion
+
+
+class Knight:
+    def __init__(self, name: str, power: int, hp: int) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+
+
+def knight_create(knights_in_dictionary: dict, knight_name: str) -> Knight:

```

Comment:
You could make this function a `classmethod` that returns the newly created `Knight` instance for the `Knight` class. Besides that, rename it to `create_knight` to be more descriptive and make an accent on the action.